vizro-core/CHANGELOG.md

```diff
@@ -11,6 +11,41 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.1.5'></a>
+
+# 0.1.5 — 2023-10-26
+
+## Removed
+
+- Remove warning message if not all registered pages are used in `Navigation` ([#117](https://github.com/mckinsey/vizro/pull/117))
+
+## Added
+
+- Add plotly layout template for waterfall chart type ([#106](https://github.com/mckinsey/vizro/pull/106))
+- Add CSS default styling for `textarea` ([#106](https://github.com/mckinsey/vizro/pull/106))
+
+- Provide ID to unique outer HTML divs on page ([#111](https://github.com/mckinsey/vizro/pull/111))
```

Comment:
I actually don't know, but does it matter if we have spaces in between or does it render properly?

```suggestion
- Add CSS default styling for `textarea` ([#106](https://github.com/mckinsey/vizro/pull/106))
- Provide ID to unique outer HTML divs on page ([#111](https://github.com/mckinsey/vizro/pull/111))
```